### ISSUE TITLE
BUG: Autostart (non-flatpack!) and start in background do not work

### ISSUE THREAD

**Issue by 3urobeat** [StreamController/StreamController#214]
**Describe the bug**
Enabling autostart in the StreamController settings has no effect, the application needs to be started manually.  
This is caused by the software assuming it was installed as a flatpak, which it was not. Therefore the flatpak run command which was added to the autostart list has no effect.

~~I then tried to just add `streamcontroller -b` to my autostart list manually, but jokes on me, `-b` has no effect and the GUI always pops up.~~  
The only error messages in the log (nothing background related as it seems to me) are:
```
ERROR    | __main__:quit_running:205 - org.freedesktop.DBus.Error.ServiceUnknown: The name is not activatable
ERROR    | src.backend.GnomeExtensions:connect_dbus:31 - Failed to connect to D-Bus: org.freedesktop.DBus.Error.ServiceUnknown: The name is not activatable
ERROR    | src.backend.WindowGrabber.WindowGrabber:init_integration:59 - Unsupported environment: kde with server: wayland for window grabber.
```

~~On a side note, could you please add the flags `-h` & `--help` which output information about the binary and its available flags instead of starting the application? It's a very irritating behavior.~~

I'm on EndeavourOS (Arch) with KDE Wayland and StreamController was installed through the AUR.

&nbsp;

**To Reproduce**
1. Install something Arch based
2. Install StreamController as something non-flatpack (e.g. AUR: `yay -S streamcontroller`)
3. Enable autostart in the StreamController settings
4. Check the autostart list (e.g. in the KDE Systemsettings application) and observe that a flatpack command was added

~~Repeat 1 & 2, then:~~

3. ~~Run `streamcontroller -b` in your terminal~~
4. ~~Observe the application starting in the foreground~~

&nbsp;

**Expected behavior**
- Add a "normal" startup command to the autostart list
- ~~Respect the background flag~~
- ~~Output help instead of starting when running with `-h` or `--help`~~

&nbsp;

Thank you!

PS: Copy pasting buttons worked for me once in the very first session and have never worked since. CTRL+C & V do nothing anymore and right click copy makes the selection jump to the next button on the right. Is this something you know of?  
Also, the OBS plugins just never manages to connect and throws the error `AttributeError: 'NoneType' object has no attribute 'get_connected'`

&nbsp;

Edit: After more investigation, I have noticed that both issues related to flags are caused by the binary created by the AUR PKGBUILD not passing flags through to the application. I will make a comment on the AUR page instead to get this resolved.

**Comment by jfbauer432** [comment]
> Also, the OBS plugins just never manages to connect

I noticed that myself a while back.  Worked around it by creating my own flatpak.

**Comment by 3urobeat** [comment]
Uhh that sucks  
I have no flatpack anything on my system and didn't plan on installing it, hopefully this can be fixed properly instead of relying on flatpacks to function

Edit: Just now realized flatpak is not written with ck, whoops

**Comment by 3urobeat** [comment]
Soo, I have quickly taken a look at the setup autostart code. It seems to correctly detect that it is not running as a flatpak, but then copies a `.desktop` file into autostart which contains a flatpack command. Am I missing something?

Just checked in a VM and the very same autostart command is added when started as a flatpak instead.

**Comment by 3urobeat** [comment]
@jfbauer432 Just wanted to let you know that the fix venv PR above fixes the OBS plugin for non-flatpak installations. It now connects for me without issue and allows switching scenes, saving replay buffer, etc.

**Comment by Core447** [comment]
The connection stuff is now fixed with #219 - Thank you!

**Comment by 3urobeat** [comment]
Fix in #218 was merged, closing
